src/codeu/chat/client/simplegui/ChatSimpleGui.java

```diff
@@ -32,7 +32,7 @@
 
     private final static Logger.Log LOG = Logger.newLog(ChatSimpleGui.class);
 
-    private JFrame mainFrame;
+    public static JFrame mainFrame;
```

Comment:
why public? and why static?

Unless you DEFINITELY need to modify the mainframe elsewhere, this should be private and create a different method called getMainFrame() which will return the mainFrame